docs/backend/guides/tget/index.md

```diff
@@ -0,0 +1,29 @@
+---
+id: index
+sidebar_position: 1
+slug: /backend/guides/tget
+---
+
+# TGET
+
+TGET stands for Transportation Global Edit Table. Each service has their own set of valid accounting codes, also known as TAC (transportation accounting codes).
+
+An example of this would be where the TOO (Transportation Ordering Officer) enters a TAC before approving a shipment. A TAC determines who pays for the shipping service (from which account). This TAC needs to be verified as a valid TAC. The MilMove application uses these to make sure that a customers order is valid. However, it should be noted that these codes do change and are updated.
+
+## Context
+
+The current implmentation is based from a sample file that had been parsed to retrieve all the unique TAC values. A new table in the database was made called `transportation_accounting_codes` and a secure migration was done to put all those unique values into the database.
```

Comment:
A question that I had while reading this was: do we know how the sample file was parsed? Was this done programmatically or manually?

Reply:
cc: @felipe-lee In case you know (it looks like you had previously worked on this a while back)

Reply:
It was parsed using a Python script. I don't know if we have that Python script up somewhere though?

Reply:
I put the script as a [comment on the jira ticket](https://dp3.atlassian.net/browse/MB-6537?focusedCommentId=26308) 🔒 because it was meant to be a one-off at the time, and it was incomplete in that (as noted in the file docstring), I'd already cleaned up the file a bit with a regex before running the script on it. We'd likely want a golang version if we automated this anyway to match the rest of the server code.